test_runner/regress/test_disk_usage_eviction.py

```diff
@@ -887,8 +889,8 @@ def test_secondary_mode_eviction(eviction_env_ha: EvictionEnv):
         readback_conf = ps_secondary.read_tenant_location_conf(tenant_id)
         log.info(f"Read back conf: {readback_conf}")
 
-        # Request secondary location to download all layers that the attached location has
-        ps_attached.http_client().tenant_heatmap_upload(tenant_id)
+        # Request secondary location to download all layers that the attached location indicated
+        # in its heatmap
         ps_secondary.http_client().tenant_secondary_download(tenant_id)
 
     # Configure the secondary pageserver to have a phony small disk size
```

Comment:
This test really shouldn't use statvfs mock -- it has nothing to do with statvfs nor is the periodic task doing us any favors; this will later be a time sink when fixing or implementing something else.

`test_fast_growing_tenant` is the closest (scrolling up) example of using the on-demand disk usage eviction, which would be less lines, and a synchronous operation.

These mock statvfs tests have an inherent fight against a background task, I think the `relieved_log_message` from this test could come during the downloads already, so when we do the `env.timelines_du` we are not certain in which phase are these fighting processes. I don't know how the detaching business really helps, but perhaps I did not see the same flakyness runs.